bugshaker/src/main/res/layout/activity_main.xml

```diff
@@ -209,6 +216,15 @@
         android:layout_marginTop="3dp"
         android:layout_weight="1"></com.github.stkent.bugshaker.flow.widget.DrawingView>
 
+    <EditText
+        android:id = "@+id/textEditing"
```

Comment:
does this really do anything? I saw we are creating an edittext using our code too?